app/services/merchants_service.rb

```diff
@@ -0,0 +1,13 @@
+class MerchantsService
+  class << self
+    def conn
+      conn = Faraday.new(url: 'http://localhost:3000/api/v1')
```

Comment:
Make your connection to just the domain, no endpoints. `/api/v1` is technically part of the endpoint URI.